python/ray/util/collective/collective.py

```diff
@@ -228,6 +228,111 @@ def barrier(group_name):
     g.barrier()
 
 
+def reduce(tensor,
+           dst_rank: int = 0,
+           group_name: str = "default",
+           op=types.ReduceOp.SUM):
+    """
+    Reduce the tensor across the group to the destination rank.
```

Comment:
for each of these methods, can you be sure to write the docstring in a way such that the first line has 1 sentence that does not have line-wrap? for ex:

```
"""Reduce the tensors across group to the destination rank.

Args:...
```

Reply:
done